src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let stateCopy = Object.assign({}, state);
```

Comment:
Consider using const instead of let without extra need

Reply:
Fixed that